physionet-django/project/managers/publishedproject.py

```diff
@@ -48,4 +51,18 @@ def accessible_by(self, user):
                 contributor_review_with_access | credentialed_with_dua_signed
             )
 
+        # Event related logic
+        # get all projects that are accessible by user through Events(Used by `hdn-research-environment` package)
+        # get all active events that user is a participant/host of
```

Comment:
I'm not convinced that these comments are useful (if anything, for me they are distracting). 

Perhaps having one comment at the top explaining the overall goal of the chunk would be worthwhile. The rest we can work out from the code.

Reply:
I agree about the comments. They were perhaps my way of making sure i dont miss anything(i normally write a ton of comments before code and work on the code). I will refactor all of the unnecessary docstring, comments in another PR.